otterdog/eclipse-4diac.jsonnet

```diff
@@ -53,8 +53,13 @@ orgs.newOrg('eclipse-4diac') {
       allow_update_branch: false,
       default_branch: "release",
       delete_branch_on_merge: false,
+      dependabot_security_updates_enabled: true,
+      has_discussions: true,
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
This is in general not needed as you can specify which permissions a workflow should be granted. We understand this its quite some work to migrate existing workflows that rely on write defaults to read, but for new workflows / projects read should be the default imho.

Reply:
I missed your comment. Yes in parallel other contributors found this solution. thx for letting me know.